tests/plans/test_export.py

```diff
@@ -87,11 +91,11 @@ def test_reply_to_mixin(plan_factory, project_factory,
     project_1 = project_factory()
     project_2 = project_factory()
     plan = plan_factory(
-        contact_name='',
+        contact_name='Joe',
```

Comment:
Jane! :)

Reply:
Or actually this is the place to put in your name to leave some marks on the software. :)

Reply:
Shoot! Next time :)